pod.go

```diff
@@ -383,6 +390,16 @@ func createPod(podConfig PodConfig) (*Pod, error) {
 		state:      State{},
 	}
 
+	// Create network only if the namespace is empty in our configuration.
+	// Being empty means this is the first time we create the network for
+	// this pod.
+	if podConfig.NetworkConfig.NetworkNSPath == "" {
```

Comment:
We will always have to call `network.create()` even if `NetNSPath` is not empty, because `create()` will always have to set the neworking namespace up.